### ISSUE TITLE
Is suspend to disk or hibernate possible?

### ISSUE THREAD

**Issue by bung69** [01org/gvt-linux#198]
Hi,  

i have tied to get suspend to disk / hibernate working with GVT-D and just noticed the QEMU parameters disabling suspend and hibernate in your guide.

```
 -global PIIX4_PM.disable_s3=1 -global PIIX4_PM.disable_s4=1
 ```

is there anyway to successfully use suspend to disk?  currently my VM just hangs on suspend.

i also notice in issue https://github.com/intel/gvt-linux/issues/175  that migration is possible.  surely if migration is possible suspend to disk is also possible?

Thanks

**Comment by TerrenceXu** [comment]
Yes, suspend to disk feature is not enabled until now. And migration code is not upstream until now.

**Comment by bung69** [comment]
After a bit more experimenting i found that an Ubuntu 20.04 or 21.10 VM with suspend to disk configured and GVT-D graphics passed through will suspend to disk and resume without issue.

Fedora 35 on the other hand will  not, hanging during suspend and then failing to boot after.   I guess Ubuntu and Fedora must use different methods to handle suspend to disk?
